test/red/runtime/util_spec.js

```diff
@@ -153,11 +153,9 @@ describe("red/util", function() {
             var v = util.getMessageProperty({a:"foo"},"msg.b");
             should.not.exist(v);
         });
-        it('should throw error if property parent does not exist', function() {
```

Comment:
I can't remember if this was discussed, but this is a change in behaviour to a public api. Why does this now return undefined rather than throw an error?

Reply:
Thank you for your review.
I am sorry, this is my mistake.  I reverted the changes.